event_dispatcher.rst

```diff
@@ -322,6 +322,18 @@ its name:
 
     $ php bin/console debug:event-dispatcher kernel.exception
 
+For the :doc:`new experimental Security </security/experimental_authenticators>`
+an event dispatcher per firewall was added. You can get the registered listeners
+for a particular event dispatcher by using the `--dispatcher` option:
```

Comment:
```suggestion
for a particular event dispatcher by using the ``--dispatcher`` option:
```

Reply:
Thanks! I keep forgetting to not use the Slack version. 😅